assets/js/admin/course-index.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * WordPress dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+
+domReady( () => {
+	jQuery( '.sensei-show-more' ).on( 'click', function ( event ) {
```

Comment:
We tend to favour arrow functions.

Reply:
Fixed in 94bde2dc911cbeebf64cb125aa2303e78af50fbe.